--- hook-PyQt5.QtGui.py	2017-07-04 17:14:25.448642115 +0100
+++ hook-PyQt5.QtGui-styles.py	2017-07-04 17:14:15.998604747 +0100
@@ -21,6 +21,5 @@
 binaries.extend(qt_plugins_binaries('inputmethods', namespace='PyQt5'))
 binaries.extend(qt_plugins_binaries('graphicssystems', namespace='PyQt5'))
 binaries.extend(qt_plugins_binaries('platforms', namespace='PyQt5'))
-
-if is_linux:
-    binaries.extend(qt_plugins_binaries('platformthemes', namespace='PyQt5'))
+binaries.extend(qt_plugins_binaries('platformthemes', namespace='PyQt5'))
+binaries.extend(qt_plugins_binaries('styles', namespace='PyQt5'))
